packages/fields/types/Content/views/slate-walker/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "main": "dist/fields.cjs.js",
+  "module": "dist/fields.esm.js",
+  "preconstruct": {
+    "source": "../src/slate-walker"
+  }
+}
```

Comment:
What's this for?

Reply:
Whoops - that's an artifact of a bad rebase. Will remove.